### PR TITLE
Compute smoothed cpu load for containers.

### PR DESCRIPTION
**Pull request by rjnagal**

### CODE REVIEW

File: info/container.go [hunk 1]
```diff
@@ -197,6 +197,8 @@ type CpuStats struct {
 		// Unit: nanoseconds
 		System uint64 `json:"system"`
 	} `json:"usage"`
+	// Number of runnable threads x 1000.
+	// We multiply by thousand to avoid using floats, but preserving precision.
```
Comment (vishh) [inline info/container.go]: Curious: Why will the number of runnable threads be a float?

Comment (rjnagal) [inline info/container.go]: This is a smoothed avg of number of running threads. I'll update the comment to make it clear.

**Comment by vishh** [comment]
LGTM. Can you take a look at the integration test failures?

**Comment by rjnagal** [comment]
vmarmol@ said that there's some flakes he's fixing. possibly related.

**Comment by vmarmol** [comment]
The integration test failure is a flake. I need to fix that.

File: info/container.go [hunk 2]
```diff
@@ -197,6 +197,8 @@ type CpuStats struct {
 		// Unit: nanoseconds
 		System uint64 `json:"system"`
 	} `json:"usage"`
+	// Smoothed average of number of runnable threads x 1000.
+	// We multiply by thousand to avoid using floats, but preserving precision.
 	Load int32 `json:"load"`
```
Comment (vmarmol) [inline info/container.go]: Only concern I have here is that all of the above stats are instantaneous except this one. We should at least document over what time period or how this is smoothed. Idk if we should export smoothing over different intervals. As long as we do this one over a small enough one we should be okay.

Comment (vmarmol) [inline info/container.go]: Milliload? :)

Comment (rjnagal) [inline info/container.go]: Nooo. Not milliload :(

Comment (rjnagal) [inline info/container.go]: We smooth it over last 10s. If the load changes from 2 to 50 and stays there, this load value should get to 50 within 10s. The instantaneous value is very spiky and not useful for analysis. If a tool wants to do its own calculation or look for bursts, the value is still available in loadstats.NrRunning.

I can change the field to LoadAverage and add the 10s interval in comments.

Comment (vmarmol) [inline info/container.go]: I think the name change and the comment sounds good. Thanks for pointing out that we also export the instantaneous one. SGTM :)

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**